pkg/types/types_test.go

```diff
@@ -32,4 +32,8 @@ components:
       - use
       - manage
       x-seal-default-action: deny
+      x-seal-attributes:
```

Comment:
please see comment above where we use `properties` instead of `x-seal-attributes`.  let's replace all places in the code using the name of `attribute` with `property`
```
openapi: "3.0.0"
components:
  schemas:
    # maybe we can make tags built in
    tag:
      type: object
      additionalProperties: true
    products.inventory:
      type: object
      properties:
        id: 
          type: string
        name:
          type: string
        tags:
          $ref: '#/components/schemas/tag'
      x-seal-actions:
      - allow
      - deny
      x-seal-verbs:
      - inspect
      - use
      - manage
      x-seal-default-action: deny 
```

you'll want to rebase from master to pick up similar changes added to action properties that may help here.